packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -100,6 +101,10 @@ export class AtomicCategoryFacet
    */
   @Prop() public sortCriteria: CategoryFacetSortCriterion = 'occurrences';
 
+  constructor() {
+    facetStore.set(this.field, this.label);
```

Comment:
There could be multiple facets with the same field, but different Ids/labels
Btw what happened with the facetIds on atomic facets? They were added here: https://github.com/coveo/ui-kit/pull/330